crates/language-server/src/functionality/item_info.rs

```diff
@@ -0,0 +1,77 @@
+use hir::{
+    hir_def::{Attr, ItemKind},
+    span::LazySpan,
+    HirDb, SpannedHirDb,
+};
+
+pub fn get_item_docstring(item: &ItemKind, hir_db: &dyn HirDb) -> Option<String> {
```

Comment:
```suggestion
pub fn get_item_docstring(item: ItemKind, hir_db: &dyn HirDb) -> Option<String> {
```

ItemKind implements `Copy` and its size is 8 bytes. No need to pass a reference.

Reply:
Nice, thank you